src/index.ts

```diff
@@ -37,7 +37,7 @@ export const Config: Schema<Config> = Schema.intersect([
   }),
 ])
 
-const conversations = new Map<string, { messageId: string; conversationId: string }>()
+const conversations = new Map<string, Conversation>()
```

Comment:
Not really good to use type `Conversation` here, as we don't save messages in the map.
If you stick with a type, you can split them into two types, but I argue with that.

Reply:
What is your specific idea? My idea is to use the `Conversation` type here and save it in a `Map`. I can't think of any other idea at the moment.

Reply:
I think just to revert this type would be good to me.
Otherwise you should write something like:
```ts
export interface Conversation {
  messageId: string
  conversationId: string
}

export interface ConversationMessage extends Conversation {
  message: string
}
```

That would be much much redundant.

Reply:
So how do I save the chat context, which is necessary in the implementation of `variant`?

Reply:
Or

```suggestion
const conversations = new Map<string, Omit<Conversation, 'message'>>()
```

while `Omit` is a built-in type of typescript.